### DISCUSSION TITLE
Could You Port The Mod to Fabric too? (to Fabric Servers)?

### DISCUSSION THREAD

**Discussion by BernardoXD72** [Abelkrijgtalles/mojang-maps#11] [Ideas]

**Comment by samuelh2005** [comment]
This isn't a mod, its a plugin. It would be difficult to convert a plugin into a mod. However you may use [cardboard](https://github.com/CardboardPowered/cardboard) to run plugins on a Fabric server, or [Arclight](https://github.com/IzzelAliz/Arclight) to run the plugin on Forge servers.

**Comment by Abelkrijgtalles** [comment]
I would maybe make a fabric mod to more easily register roads, but won't port the whole plugin to fabric (at least not for now)

**Comment by Abelkrijgtalles** [comment]
I will convert this to a discussion, because I just created the discussions tab and issues is better to just stay as issues
